### PR TITLE
add -dialog mode (new value input only)

### PR DESCRIPTION
**Pull request by seletskiy**

### CODE REVIEW

File: source/rofi.c [hunk 1]
```diff
@@ -571,7 +573,16 @@ static gboolean startup ( G_GNUC_UNUSED gpointer data )
             g_main_loop_quit ( main_loop );
         }
     }
-    else if ( find_arg_str (  "-e", &( msg ) ) ) {
+    else if ( dialog_mode == TRUE ) {
+        // force off sidebar mode:
+        config.sidebar_mode = FALSE;
+        int retv = dmenu_switcher_dialog ();
+        if ( retv ) {
+            rofi_set_return_code ( EXIT_SUCCESS );
```
Comment (kovetskiy) [inline source/rofi.c]: `rofi_set_return_code` wut

Comment (DaveDavenport) [inline source/rofi.c]: if you cannot give normal feedback, please don't give feedback.

**Comment by DaveDavenport** [comment]
I changed some stuff in GIT (trying to fix some other stuff),
` rofi -input /dev/null -dmenu -no-fixed-num-lines`  should now show your dialog as this patch did.

**Comment by seletskiy** [comment]
@DaveDavenport: Thank you! I will close current PR then.

**Pull request closed without merging**